search-commons/src/main/java/no/unit/nva/search/common/constant/Patterns.java

```diff
@@ -61,6 +61,8 @@ public final class Patterns {
      */
     public static final String PATTERN_IS_FUNDING = "[\\w]+[:\\s]{1}.+";
 
+    public static final String PATTERN_IS_NONE_PRINTABLE_CHARACTERS = "\\p{C}";
```

Comment:
nitpick: `NOT_PRINTABLE`, `NON_PRINTABLE`,  or `UNPRINTABLE` perhaps?

Could maybe also add a comment to explain what this does, i.e. "invisible control characters and unused code points".